client/post-editor/editor-notice/index.jsx

```diff
@@ -292,6 +292,21 @@ export class EditorNotice extends Component {
 		const { siteId, message, status, onDismissClick } = this.props;
 		const text = this.getErrorMessage() || this.getText( message );
 
+		if ( message === 'updated' || message === 'published' ) {
+			return (
+				<div className={ classNames( 'editor-notice', { 'is-global': true } ) }>
```

Comment:
This looks like it duplicates the code below it

Reply:
Can you elaborate how do I separate this?